### ISSUE TITLE
Home App says Fetching snapshot before loading still.

### ISSUE THREAD

**Issue by Marfre888** [dgreif/ring#351]
Hi,

Firstly, I want to thank you for this plugin it has worked great from day 1 and I've never had any problems with it, I was so happy when you released 2way audio, as that was the feature I was most waiting for.

However, I've just updated the app, and realised that it nows says Fetching Snapshot upon loading the Home app. I have to be honest, it looks hideous, the font is off, the text is too big and does not go with the home app's design at all. I'd like to disable it, and much prefer the old snapshots as before (it used to say how long it has been since they were updated in the bottom). Is there any way to do so?

In the meantime, I have downgraded back to version 9.0.1, but that means I won't get any future updates. I really to hope there is a way to disable this feature, or at least potentially make the text an overlay on the old snapshot (so it's not on black).

Thank you for all the work you do,

Mark

**Comment by cantchooseaname8** [comment]
@Marfre888 it might help to add a screenshot of what yours looks like. It looks perfect on my end and seems to fit in fine. The font isn’t too big or anything. It might be a setting that needs to be tweaked on your phone.

**Comment by town3r** [comment]
![image](https://user-images.githubusercontent.com/12396203/83986080-2cce4b00-a901-11ea-8824-2dff0657b57c.png)

**Comment by town3r** [comment]
![image](https://user-images.githubusercontent.com/12396203/83986154-67d07e80-a901-11ea-9214-16c00b0a69f3.jpeg)

**Comment by Marfre888** [comment]
Sorry I can't provide a screenshot cause I went back to a previous version. But it looks exactly as the screenshot provided by @town3r 

There is nothing wrong with the scaling it looks fine. The aesthetic of the new tile, in my opinion, just doesn't go with the Home app (Home app being all colorful, and the video tile just being black on white text). I'd prefer to be able to disable it and live with an old snapshot for 10s than having it convert to a black video tile. 

However, this seems it is not possible to disable this feature, or at least there is no documentation of it.

**Comment by dgreif** [comment]
@Marfre888 if you are unhappy with the aesthetics, you are welcome to work up some new placeholder images that are more in-line with the HomeKit design.

I'm not gonna lie, I was pretty frustrated yesterday when you opened this issue.  I have been pouring all of my free time into this project, trying to make it as seamless and reliable as possible.  I thought the placeholders were a great improvement to the new snapshot setup given the constraints of bridged cameras (See https://github.com/dgreif/ring/wiki/Snapshot-Limitations#homebridge-limitations).  I made several tweaks to how they work over the weekend, but then ended my day with someone complaining about the _aesthetics of the font_.  This is not a commercial product, it is open source.  You are more than welcome to contribute if you see something that can be improved.

**Comment by DMBlakeley** [comment]
@dgreif, Thanks for all your efforts on this plug-in! Each update has brought improvements and is much appreciated!

**Comment by Marfre888** [comment]
@dgreif In all honesty I'm not quite sure why you are feeling attacked. I'm sorry if I did offend you, it was not my intention to do so. The plugin is perfect as I said in my initial post, and my comment wasn't intended to be a complaint.

I'm sure a lot of people will like the placeholder, but unfortunately, I don't really like it and would prefer if I could disable it, and go back to how it was before, in fact, I've already rolled back to 9.0.1. If you're gonna hold that against me so be it.

I also don't follow your reasoning that because this is open source I can't share my opinion. I have offered you a solution, but clearly you brushed over it and didn't care much about what I had to say, so I have nothing further really to contribute.

Thank you vm

**Comment by dgreif** [comment]
@Marfre888 it is completely fair to share your opinion! The hard part is that what you said felt fairly inflammatory.  

> it looks hideous, the font is off, the text is too big and does not go with the home app's design at all

I have made way too many options over the years, and every one that I add is an option that I have to maintain going forward, document, and be willing to explain to users when they get confused about why it is there.  Rather than adding a "revert back to the old way" feature just because you don't like the font I used, I'd much rather have help from someone who is clearly very detail-oriented to provide a better experience to all users without them needing to find some obscure option.  If you have the tools/skills, I would happily accept some alternative placeholders that are more in line with the HomeKit design scheme.

**Comment by Marfre888** [comment]
@dgreif Ahh sorry about that, I didn't realise I wrote it was hideous, that was out of line. I was trying to revert back to 9.0.1 when I wrote that, and I was getting frustrated cause the installer was running through but not actually rolling back.

It's not necessarily the text that is bad, but more the styling.

![IMG_2195](https://user-images.githubusercontent.com/63603736/84068595-7b471c80-a9c9-11ea-87dd-b484aa9912b9.jpeg)

As I'm sure you are aware, this is how Apple handles a no response. Would it be possible to replicate something like this, though I'm not sure what icon would go well there
